code/datums/status_effects/debuffs/rust_corruption.dm

```diff
@@ -0,0 +1,16 @@
+/datum/status_effect/rust_corruption
+	alert_type = null
+	id = "rust_turf_effects"
+	tick_interval = 2 SECONDS
+	remove_on_fullheal = TRUE
+	status_type = STATUS_EFFECT_REFRESH
```

Comment:
Using `STATUS_EFFECT_REFRESH` is redundant here, because you've have it set to the default status effect duration - infinite duration. Meaning refreshing it does nothing.